InternetRamen.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@InternetRamen"
+project: Handwired w/ 3d hotswappable 3by2 Macro Pad
+---
+
+# Project Name
+Handwired w/ 3d hotswappable 3by2 Macro Pad
+
+## Summary
+I'm going to build a 6 key macro pad. This is basically a mini keyboard. Super excited for this build because it's my first project with hardware other than my PC. I've also wanted to build a keyboard from scratch and this is very similar, but managable.
+## Plan
+
+I'm going to build this using [this](https://www.youtube.com/watch?v=v9r5DKZLz68) guide. It does not require soldering and uses hand wiring to achieve the process.
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| JLCPCB 3D Printing   | https://jlcpcb.com/ | $21.94  |
+| JLCPCB Shipping   |  | $9.69 |
+| Diodes   | [https://www.adafruit.com/product/1957](https://www.amazon.com/Switching-Schottky-Glasversiegeltes-Kleinsignalschaltendes-Schottky-Dioden/dp/B09FFC7FTG/) | $11.06  |
+| Shipping for Diodes   |  | $4.54 |
+| Pro Micro Microcontroller   | https://www.amazon.com/Hosyond-ATmega32U4-Development-Micro-Controller-Compatible/dp/B09C5H78BP/ | $18.99  |
+| Wire Cutters   |  https://www.amazon.com/IGAN-P6-Precision-Clippers-Spring-loaded-Handmade/dp/B087P191LP/ | $7.99  |
+| Copper Metal Beading Wire   |  https://www.amazon.com/dp/B08687BMYV/ | $8.98  |
```

Comment:
Wanted to double check– the budget includes stripped wires, but I wanted to check that's what you wanted rather than insulated wires that won't conduct if they cross over each other.

Reply:
Yep! The guide (and guy) I'm following seems to twist the wires to achieve some sort of row and column layout. 
![image](https://user-images.githubusercontent.com/66806100/211696543-af0a70ac-a68f-4788-bace-7886085d7a6a.png)